intercom-java/src/test/java/io/intercom/api/CompanyTest.java

```diff
@@ -0,0 +1,222 @@
+package io.intercom.api;
```

Comment:
Delighted that you're adding these tests, but I'm not actually sure that they're running in CI? Taking a look I can't see any specific suites listed 🤔

Reply:
Oh maybe they are 🤔